src/content/developers/docs/layer-2-scaling/index.md

```diff
@@ -18,6 +18,53 @@ You should have a good understanding of all the foundational topics. Implementin
 - It can be unnecessarily expensive to use blockchain applications
 - Any updates to scalability should not be at the expense of decentralization of security – layer 2 builds on top of Ethereum.
 
+## Rollups {#rollups}
+
+Rollups are solutions that bundle or "roll up" sidechain transactions into a single transaction and generate a cryptographic proof, known as a SNARK (succinct non-interactive argument of knowledge). Only this proof is submitted to the main chain.
+
+_Sidechains are Ethereum-compatible, independent blockchains._
+
+In other words, rollups mean that all state and execution is handled in sidechains – signature verification, contract execution, etc. The main Ethereum chain only stores transaction data.
+
+Rollup solutions require relayers who have staked a bond in the rollup contract. This incentivises them to relay rollups accurately.
+
+**Useful for:**
+
+- reducing fees for users
+- open participation
+- fast transaction throughput
+
+There are two types of rollups:
+
+- Zero knowledge – can only handle simple transactions
+- Optimistic – can run anything Ethereum can using an optimistic version of the EVM, the Optimistic virtual machine
```

Comment:
I think the primary differentiation between ZK & optimistic rollups is the proof mechanism, i.e. validity proofs vs. fraud proofs: 
- https://support.deversifi.com/en/docs/77-what-is-the-difference-between-zk-rollup-and-optimistic-rollup
- https://medium.com/starkware/validity-proofs-vs-fraud-proofs-4ef8b4d3d87a
- https://medium.com/starkware/validity-proofs-vs-fraud-proofs-strike-back-4d0bf90eed15

From my understanding, it's these security models that creates the core trade-offs between these 2 approaches.

Reply:
I didn't include content about ZK-rollups in the scope of this update as I only did the research on Optimistic. If we think it's important context for understanding Optimistic we can add more info on ZK-rollups before merging.

Reply:
I think we should at least explain the difference in terms of validity proof vs. fraud proof. As mentioned elsewhere, it's possible that optimistic rollup implementations `can only handle simple transactions` as well, so I don't think that's a good way to define the 2 types of rollups.